dftimewolf/lib/collectors/aws_logging.py

```diff
@@ -27,17 +27,20 @@ def __init__(self,
     self._profile_name: Optional[str] = None
     self._query_filter: Optional[str] = None
     self._start_time: Optional[datetime] = None
-    self._end_time:Optional[datetime] = None
+    self._end_time: Optional[datetime] = None
+    self._region: str = None  # type: ignore
```

Comment:
Why is type checking disabled here instead of using `Optional`?

Reply:
The param is not optional, and needs to be set, but that value only comes later in `SetUp`.